server/server.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Main application file
+ */
+
+'use strict';
+
+// Set default node environment to development
+// process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+
+var express = require('express');
+var mongoose = require('mongoose');
+// var config = require('./config/environment');
+
+// Connect to database
+mongoose.connect('mongodb://localhost/telebum');
+// mongoose.connect(config.mongo.uri, config.mongo.options);
+
+// Populate DB with sample data
+// if(config.seedDB) { require('./config/seed'); }
+
+// Setup server
+var app = express();
+var server = require('http').createServer(app);
+require('./config/express')(app);
```

Comment:
move these to the top of the file, use `var expressConfig = require('./config/express')` and then `expressConfig(app)`